server/routes/river-and-sea-levels.js

```diff
@@ -70,6 +91,8 @@ module.exports = [{
         'river-id': joi.string(),
         'target-area': joi.string(),
         types: joi.string().allow('S', 'M', 'C', 'G', 'R'),
+        'rloi-id': joi.string(),
+        d: joi.string(),
```

Comment:
If we now get the station from the cached geojson then the direction parameter is now unnecessary as if a station is an M with upstream downstream the coordinates are the same for u or d.